tools/metadata_loader/main.py

```diff
@@ -0,0 +1,185 @@
+import argparse
+import json
+import os
+from urllib.parse import quote
+
+import requests
+from azure.storage.blob import (  # pyright: ignore[reportMissingImports]
+    BlobServiceClient,
+)
+from dotenv import load_dotenv
+
+
+def get_config():
+    load_dotenv()
+    return {
+        "API_URL_BASE": os.environ.get("API_URL_BASE"),
```

Comment:
I normally rather use os.getenv in case you do not wwant to test if the env exist.

Reply:
ok